gulpfile.js

```diff
@@ -0,0 +1,38 @@
+var eol = require('gulp-eol');
```

Comment:
Please use https://github.com/netzstrategen/gulp-task-collection instead of custom gulpfile. 🙏